docs/sources/developers/plugins/add-support-for-variables.md

```diff
@@ -58,3 +58,97 @@ For data sources, you need to use the [getTemplateSrv]({{< relref "../../package
      return { data };
    }
    ```
+
+## Add support for variable queries to your data source
+
+[Query variables]({{< relref "../../variables/variable-types/add-query-variable/" >}}) is a type of variable that allows you to query a data source for the values. By adding support for variable queries to your data source plugin, users can create dynamic dashboards based on data from your data source.
```

Comment:
```suggestion
[Query variables]({{< relref "../../variables/variable-types/add-query-variable/" >}}) are allows you to query a data source for values. By adding support for variable queries to your data source plugin, users can create dynamic dashboards based on data from your data source.
```

Reply:
Do we want to mention that this would be necessary for Repeated panels/rows and perhaps link to that topic as well?

Reply:
Hm, I don't think this piece needs to explain the different use cases for variables. 

Did you mean `Query variables are _variables that_ allow you ...`?

Reply:
I actually meant "Query variables allow you to..." but your way works as well.